docs/making-a-release.md

```diff
@@ -70,6 +69,38 @@ Tag the head of your release branch with the release tag, and push
 % git push --tags
 ```
 
+## Updating the `:latest` tag
+
+If you've cut a new production-ready release, you'll need to update `:latest` on Docker Hub as well.
+
+Firstly, you'll need to be:
+
+- logged in to Docker Hub
+- A member of the projectcontour Docker Hub org with push rights.
+
+Once that's all true, do the following steps:
+
+```shell
+$ docker login
+Authenticating with existing credentials...
+Login Succeeded
+# This is the version you want to make latest
+$ export LATEST_VERSION=v1.0.0
```

Comment:
top tip: env LATEST_VERSION=v1.0.0 make tag-tatest will avoid having to unset that env var later.

Reply:
Good point.